docs/StardustDocs/topics/gettingStartedJupyterNotebook.md

```diff
@@ -0,0 +1,35 @@
+[//]: # (title: Get started with Kotlin DataFrame on Jupyter Notebook)
+
+## Jupyter Notebook
+
+You can use the Kotlin DataFrame library in Jupyter Notebook and in Jupyter Lab.
```

Comment:
notebooks

Reply:
Both of them are product names, and we mean these products, not just notebooks. If only notebook is mentioned, but Jupyter notebook looks weird like jupyter Notebook

Reply:
If you're referring to the product, then indeed "Jupyter Notebook". But how often do you say it out loud like that in a sentence? Sounds like an Apple presentation if you say "You can run your Jupyter Notebook (TM) however you like!", compared to "You can run any of your Jupyter notebooks however you like".
I think more often, you are referring to notebooks the concept than the actual name.